### ISSUE TITLE
Error in downloading model from huggingface

### ISSUE THREAD

**Issue by Wesady** [evo-design/evo#94]
HI
When I download model from huggingface
```
from transformers import AutoConfig, AutoModelForCausalLM

model_name = 'togethercomputer/evo-1-8k-base'
model_config = AutoConfig.from_pretrained(model_name, trust_remote_code=True, revision="1.1_fix")
model_config.use_cache = True

model = AutoModelForCausalLM.from_pretrained(
    model_name,
    config=model_config,
    trust_remote_code=True,
    revision="1.1_fix"
)
```
I got the error Could not locate the togethercomputer/evo-1-131k-base--configuration_hyena.py inside togethercomputer/evo-1-8k-base.
My transformers version is `4.27.0`.
Can you give me some help? Thank you very much!

**Comment by GoldenJin24** [comment]
so sad,i met the same question.I download the model files in huggingface,then offline loading the model 1.5-8k like below code

**Comment by GoldenJin24** [comment]
lucky!althougth i don't solve this problem,in chinese i uesd
'''
#bash like this
HF_ENDPOINT=https://hf-mirror.com    python    your_script.py

#code like readme.txt

from transformers import AutoModelForCausalLM, AutoTokenizer

from evo import Evo
import torch

device = 'cuda:0'

evo_model = Evo('evo-1.5-8k-base')
model, tokenizer = evo_model.model, evo_model.tokenizer
model.to(device)
model.eval()

print('================ start inference')
sequence = 'ATCG'
input_ids = torch.tensor(
    tokenizer.tokenize(sequence),
    dtype=torch.int,
).to(device).unsqueeze(0)

with torch.no_grad():
    logits, _ = model(input_ids) # (batch, length, vocab)

print('Logits: ', logits)
print('Shape (batch, length, vocab): ', logits.shape)

'''
sucessed!
